<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CampoTecyAdmin</class>
 <widget class="QDialog" name="CampoTecyAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="checkBox_jefe">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Jefe de Obra</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_resi">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Residente frente 1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_chof">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Chofer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_vela">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Velador</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_secre">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Secretaria</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_jefe">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_resi">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_chof">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_vela">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_secre">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Monto Bruto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_agregar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>+ Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_regresar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- Regresar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
